index.html

```diff
@@ -283,6 +283,14 @@
       default: crickets.start(false); break;
     }
   },
+
+  // Direct message in the hackiest possible way: send the message
+  // to everyone, and all of the non-targets ignore it.
+  msg: function(obj) {
+    var command = obj.message.match(/^\/msg\s+(\S+)\s+(.*)/);
+    if (!command || (command[1] !== nick.value)) { return " "; }
+    return false;
```

Comment:
This should return something styled.   Suggest:
```
return "<i>"+ nick +" (private) : "+ command[2] +"</i></br>"
```

Plus alter the regex above to also match the message.